src/main/kotlin/blackjack/domain/BlackjackGame.kt

```diff
@@ -0,0 +1,28 @@
+package blackjack.domain
+
+import blackjack.domain.player.Player
+import blackjack.domain.player.Players
+
+class BlackjackGame(
+    private val deck: Deck,
+    val players: Players,
+) {
+    fun start() {
+        players.drawInitCards(deck)
+    }
+
+    fun isPlaying(): Boolean {
+        return players.isExistWaitingPlayer()
+    }
+
+    fun askDrawToCurrentTurnPlayer(isDrawCard: Boolean) {
+        val player = players.findCurrentTurnPlayer()
```

Comment:
players를 붙여주지 않아도 괜찮아보여요!


```suggestion
        val player = findCurrentTurnPlayer()
```